index.html

```diff
@@ -10,18 +10,44 @@
   <body>
     <div class="container">
       <header>
-        <h1 class="logo">Riyana Abdi</h1>
-        <img src="SVG/Icon - Menu.svg" alt="" class="menuIcon" />
-        <ul class="menuList">
-          <li>Portfolio</li>
-          <li><a href="#about">About</a></li>
-          <li><a href="#Contact">Contact</a></li>
-          <li>
-            <a href="mailto:karanjain2212@gmail.com"
+       <nav class="navbar">
+        <h1 class="logo">Riyana </h1>
+        <div class="humburger">
+          <img src="./img/Menu.png" alt="" id="openBtn">
+        </div>
```

Comment:
[OPTIONAL]
- [ ] Kindly try to follow the design on Figma as much as possible. the hamburger button should be align to the right instead of center 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/196249389-24ea00b1-f07e-4002-ba1c-329dc92d4b57.png)
 
## Figma design: 
![image](https://user-images.githubusercontent.com/50721479/196249654-65a997bb-6a3b-4302-a24b-8ea5eb138541.png)